cypher/cypher-docs/src/docs/dev/syntax/naming.asciidoc

```diff
@@ -5,22 +5,30 @@ We describe here rules and recommendations for the naming of node labels, relati
 
 == Naming rules
 
-* Must begin with an alphabetic letter.
+* Alphabetic letters:
+** Names should begin with an alphabetic letter.
 ** This includes "non-English" characters, such as `å`, `ä`, `ö`, `ü` etc.
-** If a leading non-alphabetic character is required, use backticks for escaping; e.g. `++`^n`++`.
-* Can contain numbers, but not as the first character.
+//** If a leading non-alphabetic character is required, use backticks for escaping; e.g. `++`^n`++`.
+* Numbers:
+** Names should not begin with a number without escaping.
 ** To illustrate, `1first` is not allowed, whereas `first1` is allowed.
-** If a leading numeric character is required, use backticks for escaping; e.g. `++`1first`++`.
-* Cannot contain symbols.
+//** If a leading numeric character is required, use backticks for escaping; e.g. `++`1first`++`.
+* Symbols:
+** Names cannot contain symbols unless these are escaped.
```

Comment:
Names cannot contain symbols except for underscore, e.g, `my_variable`, and `$` as the first character to denote a parameter, as given by `$myParam`.